simple/deploy.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -e
+
+ECHO=`which echo`
+KUBECTL=`which kubectl`
+
+# methods
+function echoBold () {
+    ${ECHO} -e $'\e[1m'"${1}"$'\e[0m'
+}
+
+function usage () {
+    echoBold "This script automates the installation of WSO2 Identity Server with Analytics Kubernetes resources\n"
+    echoBold "Allowed arguments:\n"
+    echoBold "-h | --help"
+    echoBold "--wu | --wso2-username\t\tYour WSO2 username"
+    echoBold "--wp | --wso2-password\t\tYour WSO2 password"
+    echoBold "--cap | --cluster-admin-password\tKubernetes cluster admin password\n\n"
+}
+
+WSO2_SUBSCRIPTION_USERNAME=''
+WSO2_SUBSCRIPTION_PASSWORD=''
+ADMIN_PASSWORD=''
+
+# capture named arguments
+while [ "$1" != "" ]; do
+    PARAM=`echo $1 | awk -F= '{print $1}'`
+    VALUE=`echo $1 | awk -F= '{print $2}'`
+
+    case ${PARAM} in
+        -h | --help)
+            usage
+            exit 1
+            ;;
+        --wu | --wso2-username)
+            WSO2_SUBSCRIPTION_USERNAME=${VALUE}
+            ;;
+        --wp | --wso2-password)
+            WSO2_SUBSCRIPTION_PASSWORD=${VALUE}
+            ;;
+        --cap | --cluster-admin-password)
+            ADMIN_PASSWORD=${VALUE}
+            ;;
+        *)
+            echoBold "ERROR: unknown parameter \"${PARAM}\""
+            usage
+            exit 1
+            ;;
+    esac
+    shift
+done
+
+# create a new Kubernetes Namespace
+${KUBECTL} create namespace wso2
+
+# create a new service account in 'wso2' Kubernetes Namespace
+${KUBECTL} create serviceaccount wso2svc-account -n wso2
+
+# switch the context to new 'wso2' namespace
+${KUBECTL} config set-context $(${KUBECTL} config current-context) --namespace=wso2
+
+# create a Kubernetes Secret for passing WSO2 Private Docker Registry credentials
+${KUBECTL} create secret docker-registry wso2creds --docker-server=docker.wso2.com --docker-username=${WSO2_SUBSCRIPTION_USERNAME} --docker-password=${WSO2_SUBSCRIPTION_PASSWORD} --docker-email=${WSO2_SUBSCRIPTION_USERNAME}
+
+echoBold 'Finished'
```

Comment:
I think it would be better if we include the deploying command within the deploy.sh script itself. Here we are saying "Finish" and ask the user to execute another command right? WDYT?.